GNUmakefile

```diff
@@ -19,23 +19,23 @@ test: fmtcheck
 testacc: fmtcheck
 	TF_ACC=1 go test $(TEST) -v $(TESTARGS) -timeout 120m
 
-vet:
-	@echo "go vet ."
-	@go vet $$(go list ./... | grep -v vendor/) ; if [ $$? -eq 1 ]; then \
-		echo ""; \
-		echo "Vet found suspicious constructs. Please check the reported constructs"; \
-		echo "and fix them if necessary before submitting the code for review."; \
-		exit 1; \
-	fi
-
 fmt:
-	gofmt -w $(GOFMT_FILES)
+	@echo "==> Fixing source code with gofmt..."
+	gofmt -s -w ./$(PKG_NAME)
 
+# Currently required by tf-deploy compile
 fmtcheck:
-	@sh -c "'$(CURDIR)/scripts/gofmtcheck.sh'"
+	@echo "==> Checking source code against gofmt..."
+	@gofmt -l -s ./$(PKG_NAME)
```

Comment:
I think the reason this was in a script though was that the `gofmt` exit code doesn't actually fail if its not formatted.

Reply:
You're correct -- fixed by reinstating the old script for now.

```
$ gofmt -l -s ./aws
aws/resource_aws_api_gateway_domain_name.go
$ echo $?
0
```